### ISSUE TITLE
[9.5] creation of ticket on helpdesk - fatal error on refresh

### ISSUE THREAD

**Issue by btry** [glpi-project/glpi#6988]
Hi

### to reproduce

1 - set a ticket template with category as mandatory field
2 - ensure you have a ticket category available in GLPI
3 - on simplified interface open the ticket creation form
4 - select a category (the form refreshes)
5 - observe the fatal error

### backtrace

```
[2020-02-28 11:48:38] glpiphplog.CRITICAL:   *** Uncaught Exception Error: Call to a member function getMandatoryMark() on string in /home/tbugier/public_html/glpi-95/inc/commonitilobject.class.php at line 4145
  Backtrace :
  inc/ticket.class.php:3982                          CommonITILObject->showActorAddFormOnCreate()
  inc/ticket.class.php:3864                          Ticket::showFormHelpdeskObserver()
  front/tracking.injector.php:90                     Ticket->showFormHelpdesk()
  
```

### screenshot 

![image](https://user-images.githubusercontent.com/14139801/75541831-db1f4600-5a1e-11ea-8ef9-b4ded7df1868.png)

**Comment by trasher** [comment]
As far as I've seen, this issue does not exists in 9.4 because we do not have `_tickettemplates_id` in `$options`; event if the hidden field was present in the original form.
But I've no idea why right now :/

**Comment by cedric-anne** [comment]
Fixed by 1b65621.
